lib/util.js

```diff
@@ -125,6 +125,11 @@ util.isArrayBufferView = function(x) {
   return x && util.isArrayBuffer(x.buffer) && x.byteLength !== undefined;
 };
 
+// Node versions >= 10.2 support native keyPair generation,
```

Comment:
Docs say it's 10.12.0:
https://nodejs.org/dist/latest-v10.x/docs/api/crypto.html#crypto_crypto_generatekeypair_type_options_callback

Reply:
Woof. Good catch. Will update.

Reply:
👋